### PR TITLE
Add Sym Class and Functionality

### PR DESCRIPTION
**Pull request by Arunsp2000**

### CODE REVIEW

**Comment by nitesh31mishra** [comment]
Approved.

**Review by nitesh31mishra — APPROVED** [review]

**Review by nagarajumadamshetti — APPROVED** [review]
Reviewed

File: src/sym.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import math
+
+'''
+Class Sym which handles symmbolic characters 
+'''
+class sym:
```
Comment (agupta15k) [inline src/sym.py]: Class name should be starting with a capital letter as per the [generic naming convention in Python](https://peps.python.org/pep-0008/#class-names)

**Review by agupta15k — COMMENTED** [review]
Not a breaking change, but please try to follow a standard while writing functions, there are places where assignment using `=` has leading and trailing spaces, while there are no spaces at other places. There are new line characters within functions, which can hamper the readability a bit.

**Review by agupta15k — APPROVED** [review]
Approved

**Pull request merged**